### ISSUE TITLE
use ctime to recover mtime

### ISSUE THREAD

**Issue by antoine-mesobfc** [nextcloud-gmbh/mtime_fixer_tool_kit#27]
I found more than 60000  files with mtime < 86400 in my instance (SQL count).

solvable_files.sh  script is very slow, it has been running for 4 hours in "list mode" and it is not finished, but maybe I can slow it down again :

Why not use "ctime" to correct "mtime" in database?
The last change date is often the same as the last modification date (metadata are rarely changed).

This might avoid losing some data by entering the current date in the ctime.
Am I wrong ?

**Comment by platima** [comment]
Yeah I would MUCH prefer to keep any dates possible. This script does not feel like a 'fix'

**Comment by platima** [comment]
Implemented in #32
